apps/tlon-web/e2e/shipManifest.json

```diff
@@ -23,13 +23,24 @@
   },
   "~ten": {
     "authFile": "e2e/.auth/ten.json",
-    "downloadUrl": "https://bootstrap.urbit.org/rube-ten10.tgz",
+    "downloadUrl": "https://storage.googleapis.com/tlon-messenger-public-assets/e2e-nodes/rube-ten11.tgz",
```

Comment:
@patosullivan I didn't know how to upload to that domain, so I was just pushing my updated piers to a gcloud bucket I had access to.

Reply:
Daniel added you and the rest of the devs on the team to the `tlon` project so you'll have access to the bootstrap.urbit.org bucket in the future.